src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -154,6 +155,12 @@ public static async Task<string> AzureActiveDirectoryAuthenticationCallback(stri
             return response.Token;
         }
 
+        /// <summary>
+        /// Enables configured 'Sql Authentication Provider' for 'Active Directory Interactive' authentication mode to be used 
+        /// when user chooses 'Azure MFA'.
+        /// </summary>
+        public static bool EnableSqlAuthenticationProvider { get; set; }

```

Comment:
the services are singleton so we generally don't statics for state

Reply:
I needed to use this in static API, that's why made it static.. will attach it to instance instead.